test/k8s-integration/driver.go

```diff
@@ -65,12 +67,18 @@ func installDriver(goPath, pkgDir, stagingImage, stagingVersion, deployOverlayNa
 
 	// TODO (#139): wait for driver to be running
 	time.Sleep(time.Minute)
-	statusCmd := exec.Command("kubectl", "describe", "pods", "-n", "default")
+	statusCmd := exec.Command("kubectl", "describe", "pods", "-n", "gce-pd-csi-driver")
 	err = runCommand("Checking driver pods", statusCmd)
 	if err != nil {
 		return fmt.Errorf("failed to check driver pods: %v", err)
 	}
-
+	time.Sleep(time.Minute)
```

Comment:
Do we need this second check?

Reply:
oh, I was thinking to keep "default" and add "gce-pd-csi-driver" namespace one. 
I am not sure why it was checking "default" before, or just a mistake.

Also this runCommand actually does not really check pod status. It will not return error if pods are not successfully running. In the second check I added, I print out the output for debugging.

Reply:
When we changed namespaces, we forgot to update the test to check in the correct namespace. So we can remove the check for default namespace.

Reply:
got it. I plan to have follow up PR to check pod status and print out debug log etc.

Reply:
I believe @mattcary is going to look into it as part of https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/139

Reply:
I just wrote https://github.com/kubernetes/kubernetes/pull/93159, could we use that here as well?

I haven't figured out where the best places to wait for things are---if it has to be done here and in k8s/k8s

Reply:
So for our pd driver repo tests, we need to add the logic in our k8s-integration test runner, which bring up the driver outside of the e2e tests.

Reply:
thanks! I think we have to put the logic here to wait for pod running.

Reply:
@jingxu97 Do you want to look at kubernetes/kubernetes#93159 and make sure it has the right parameters? In particular the StatefulSet name is hardcoded, maybe I should move that to a parameter to make it usable from the pd driver repo? (I'd need to look again in the pd driver repo to see if that's necessary).

Reply:
So this repo switched to use a Deployment instead of a StatefulSet, so importing it is not going to work as is.

Maybe we can switch k/k to use Deployment too?

Reply:
@mattcary so far your pr looks good to me. In case some pod not running correctly, do we collect logs on events which help debug?

Reply:
never mind, I think logs are collected https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/140

Reply:
Done, switched to deployment and moved the driver name to the exported function.

Reply:
Actually there already is a deployment-waiter in  framework/deployment/wait that could be used. That's what my change is using now, so there's nothing to export to you.